webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -0,0 +1,160 @@
+};
+
+export const ErrataSeverity = ({ severity }) => {
+  let color;
+  let label;
+
+  switch (severity) {
+    case 'Moderate':
+      color = 'yellow';
+      label = __('Moderate');
+      break;
+    case 'Important':
+      color = 'orange';
+      label = __('Important');
+      break;
+    case 'Critical':
+      color = 'red';
+      label = __('Critical');
+      break;
+    default:
+      label = __('N/A');
+  }
+  return <TableText wrapModifier="nowrap">{color && <SecurityIcon color={color} title={label} />} {label} </TableText>;
+};
+
+ErrataSeverity.propTypes = {
+  severity: PropTypes.string.isRequired,
+};
+
+export const ErrataTab = () => {
+  const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
+  const { id: hostId } = hostDetails;
+
+  const emptyContentTitle = __('This host currently does not have any installable errata for this content view.');
+  const emptyContentBody = __('Installable Errata will appear here when available.'); // needs link
+  const emptySearchTitle = __('No matching Installable Errata found');
+  const emptySearchBody = __('Try changing your search settings.');
+  const [searchQuery, updateSearchQuery] = useState('');
+  const columnHeaders = [
+    __('Errata'),
+    __('Type'),
+    __('Severity'),
+    __('Synopsis'),
+    __('Published Date'),
+  ];
+
+  const fetchItems = useCallback(
+    params => getInstallableErrata(hostId, params),
+    [hostId],
+  );
+
+  const response = useSelector(state => selectAPIResponse(state, HOST_ERRATUM_KEY));
+  const { results, ...metadata } = response;
+  const status = useSelector(state => selectHostErrataStatus(state));
+
+  return (
+    <TableWrapper
+      {...{
+          metadata,
+          emptyContentTitle,
+          emptyContentBody,
+          emptySearchTitle,
+          emptySearchBody,
+          searchQuery,
+          updateSearchQuery,
+          status,
+          }}
+      fetchItems={fetchItems}
+      autocompleteEndpoint={`/hosts/${hostId}/errata/auto_complete_search`}
+      foremanApiAutoComplete
+      variant={TableVariant.compact}
+    >
+      <Thead>
+        <Tr>
+          {columnHeaders.map(col =>
+            <Th key={col}>{col}</Th>)}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {results?.map((erratum) => {
+            const {
+              id,
+              errata_id: errataId,
+              created_at: createdAt,
+              updated: publishedAt,
+              title,
+            } = erratum;
+            return (
+              <Tr key={`${id}_${createdAt}`}>
+                <Td>
+                  <a href={urlBuilder(`errata/${id}`, '')}>{errataId}</a>
+                </Td>
+                <Td><ErrataType {...erratum} /></Td>
+                <Td><ErrataSeverity {...erratum} /></Td>
+                <Td><TableText wrapModifier="truncate">{title}</TableText></Td>
+                <Td key={publishedAt}><IsoDate date={publishedAt} /></Td>
+              </Tr>
```

Comment:
Should add the action menu and (disabled) action items here, at the end of each table row

Reply:
@parthaa thoughts on adding this?